README.md

```diff
@@ -1,13 +1,25 @@
-# VGCN Infrastructure Management
+# VGCN infrastructure management
 
-This repository allows for definition and management of VGNC resources in the
-bwCloud for usegalaxy.eu
+This repository defines and manages the
+[VGNC](https://github.com/usegalaxy-eu/vgcn) compute nodes in the
+[bwCloud](https://www.bw-cloud.org/) for [usegalaxy.eu](usegalaxy.eu)
```

Comment:
can we call this bwCloud and de.NBI-cloud to make our funders happy?

Reply:
Like this? I did not fully understand sorry.

```markdown
This repository defines and manages the
[Virtual Galaxy Compute Nodes](https://github.com/usegalaxy-eu/vgcn) on
[bwCloud](https://www.bw-cloud.org/)
(part of [de.NBI-cloud](https://www.denbi.de/))
for [usegalaxy.eu](usegalaxy.eu)
```

Reply:
This repository defines and manages the
[Virtual Galaxy Compute Nodes](https://github.com/usegalaxy-eu/vgcn) on
[bwCloud](https://www.bw-cloud.org/) / [de.NBI-cloud](https://www.denbi.de/)
for [usegalaxy.eu](usegalaxy.eu)